Windows/Debugger/Debugger_Disasm.cpp

```diff
@@ -32,11 +32,37 @@
 #include <windowsx.h>
 #include <commctrl.h>
 
+// Includes for Trace Logger Mod
+#include <fstream>
+#include <string>
+#include <sstream>
+
+
+
 // How long (max) to wait for Core to pause before clearing temp breakpoints.
 static const int TEMP_BREAKPOINT_WAIT_MS = 100;
 
 static FAR WNDPROC DefGotoEditProc;
 
+// Begin Trace Logger Changes
+int check = CreateDirectoryA("Trace Logs\\", NULL);
```

Comment:
Why is this in global scope? Am I missing something?

Reply:
It made sense to me to keep this in the global scope as long as the things below it are also in the global scope. The idea being I wanna be able to access file reading/writing at any point but the file it's writing to is determined by the date/time the debugger window is open, so whatever the "current" file is makes sense to keep globally, to me.

Reply:
Ah nevermind, i misunderstood what you were talking about and fixed it, once i upload it that should be fixed.